spec/models/cart_item_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+
+RSpec.describe CartItem, type: :model do
+  let(:cart_item) {create(:cart_item, pricing: pricing, quantity: quantity)}
+  let(:pricing) {}
+  let(:quantity) {10}
+
+  describe '#total_price' do
+    subject {cart_item.total_price}
+
+    context 'when there is no pricing' do
+      it {is_expected.to eq 0}
+    end
+
+    context 'when there is a pricing' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.